src/adapter/src/catalog/consistency.rs

```diff
@@ -408,3 +555,42 @@ enum ObjectDependencyInconsistency {
         object_b: GlobalId,
     },
 }
+
+#[derive(Debug, Serialize, Clone, PartialEq)]
+enum ItemInconsistency {
```

Comment:
Some of these inconsistencies seem specific to schemas or databases, not items.

Reply:
Yeah... the naming wasn't the best. This was pulled out of this PR though